docs/services/ml/_index.md

```diff
@@ -1,22 +1,24 @@
 ---
-title: "Deploy a Model"
-linkTitle: "Deploy Model"
+title: "ML Models Service"
```

Comment:
Question: "ML Models" service vs "ML Model" service

Do you think you could make this consistent across the intro/titles here?

Reply:
Ml models is what it's called in the product

Reply:
![Screenshot 2023-04-13 at 22 47 31](https://user-images.githubusercontent.com/5212232/231879316-33cc1aaa-e166-4278-a06e-b90fa83ccbe8.png)

Reply:
Oh dear. Maybe we can ping frontend and change it there